src/components/pages/FoundationHeader.astro

```diff
@@ -443,4 +448,15 @@ function isClickOutside(event: MouseEvent, nodeList: NodeListOf<Element>) {
   });
   return !clickedInsideTarget;
 }
+
+// Site sub-navigation highlighting parent menu item 
+const devLinks = document.querySelectorAll('.site-nav__links a[href*="/developers"]');
+const currentPath = window.location.href;
+devLinks.forEach( devLink => {
+ if(devLink.href === currentPath){
+  const parentItemLink = devLink.closest(".menu-item--level-1").children[0]
```

Comment:
```
const parentMenu = devLink.closest('.menu-item--level-1');
const parentMenuLink = parentMenu?.querySelector('a');
devLink.classList.add('is-active');
parentMenuLink?.classList.add('is-active');
```

Your code can break if you assume an object exists and it in fact hasn't been found. So it's better to first get the element, check it exists and didn't return null and then make the next query selector. 

It's also better to search for what you're looking for, an anchor element, not just any first child since the structure could change and then you'd be returning something unexpected here.